packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -0,0 +1,40 @@
+import {GraphQLList, GraphQLNonNull, GraphQLObjectType} from 'graphql'
+import {GQLContext} from '../graphql'
+import IntegrationProviderOAuth1 from './IntegrationProviderOAuth1'
+import TeamMemberIntegrationAuthOAuth1 from './TeamMemberIntegrationAuthOAuth1'
+
+const JiraServerIntegration = new GraphQLObjectType<any, GQLContext>({
+  name: 'JiraServerIntegration',
+  description: 'Jira Server integration data for a given team member',
+  fields: () => ({
+    auth: {
+      description: 'The OAuth1 Authorization for this team member',
+      type: TeamMemberIntegrationAuthOAuth1,
+      resolve: async ({teamId, userId}, _args, {dataLoader}) => {
```

Comment:
+1 should we use explicit types in resolvers?

Reply:
Yes, absolutely we should! Thanks for reminding!

Reply:
types missing! 😅  also, any thoughts on specifying source type in GQL like that
```
const JiraServerIntegration = new GraphQLObjectType<{teamId: string; userId: string}, GQLContext>({
  name: 'JiraServerIntegration',
``` 
instead of doing it in every resolver? 🤔

Reply:
I fixed the types below :facepalm: Yes, your way is much cleaner!